### PR TITLE
Add FancyPCA

### PR DESCRIPTION
**Pull request by BloodAxe**
Adds FancyPCA augmentation https://pixelatedbrian.github.io/2018-04-29-fancy_pca/

### CODE REVIEW

File: albumentations/augmentations/functional.py [hunk 1]
```diff
@@ -1891,3 +1891,69 @@ def multiply(img, multiplier):
...
         return _multiply_non_uint8(img, multiplier)
+
+
+def fancy_pca(img, alpha=0.1):
+    """Perform 'Fancy PCA' augmentation from:
+    http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf
+
+    Args:
+        img:  numpy array with (h, w, rgb) shape, as ints between 0-255)
+        alpha_std:  how much to perturb/scale the eigen vecs and vals
```
Comment (albu) [inline albumentations/augmentations/functional.py]: arg is alpha already

File: albumentations/augmentations/transforms.py [hunk 2]
```diff
@@ -2970,3 +2971,34 @@ def targets_as_params(self):
...
+        image
+
+    Image types:
+        3-channel uint8 images only
+
+    Credit:
+        http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf
+        https://deshanadesai.github.io/notes/Fancy-PCA-with-Scikit-Image
+        https://pixelatedbrian.github.io/2018-04-29-fancy_pca/
+    """
```
Comment (Dipet) [inline albumentations/augmentations/transforms.py]: Need add descriptions for `alpha_std` from `fancy_pca` function.

**Comment by ternaus** [comment]
Example in documentation?

File: albumentations/augmentations/transforms.py [hunk 3]
```diff
@@ -3001,6 +3002,44 @@ def get_transform_init_args_names(self):
         return "multiplier", "per_channel", "elementwise"
 
 
+class FancyPCA(ImageOnlyTransform):
+    """Augment RGB image using FancyPCA from Krizhevsky's paper
+    "ImageNet Classification with Deep Convolutional Neural Networks"
+
+    Args:
+        alpha (float):  how much to perturb/scale the eigen vecs and vals
+            the paper used std=0.1
```
Comment (Dipet) [inline albumentations/augmentations/transforms.py]: Please add that alpha is selected from the range of `[0, alpha]` using gamma distribution.

Comment (BloodAxe) [reply albumentations/augmentations/transforms.py]: Thanks, fixed

**Pull request merged**